app/utils/DataService.js

```diff
@@ -122,3 +122,7 @@ export const getService = id => get(`/api/services/${id}`)
       }
     );
   });
+
+export const addFeedback = (url, body) => (
+  post(url, body).then(res => res.json())
```

Comment:
I'd suggest that you put the two URLs (resource vs. service) here in this function rather than in FeedbackModal.jsx, since it should be the responsibility of this DataService file to understand the URL endpoints. You can still have a parameter that lets you select between resource vs. service, but as long as the FeedbackModal doesn't have to actually construct the API URLs, that keeps the responsibilities separated out.